static/scripts/alerts.js

```diff
@@ -7,7 +7,7 @@ export default function initAlerts(viewToInit) {
 	}
 	initialized = true;
 
-	const view = viewToInit === 'loggedin';
+	const view = viewToInit;
```

Comment:
why changing the type... ? view was a boolean before

Reply:
Because now we willl pass as a parameter the name of the view, before was just for loggedin now more files will be using this function.